packages/dcos-metrics/buildinfo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "requires": ["mesos", "boost-libs", "avro-cpp"],
+  "single_source" : {
+    "kind": "git",
+    "git": "https://github.com/dcos/dcos-metrics.git",
+    "ref": "b1a88554bf107f9443245e787e82db4f502b300d",
```

Comment:
Or if `ref` will accept a tag, the repo is currently tagged at `0.0.8`, which will hopefully be the last tag before 1.0-rc1.